### DISCUSSION TITLE
Availability on F-Droid

### DISCUSSION THREAD

**Discussion by itsluminous** [trakt/showly#700] [Ideas / Feature Requests]
It would be great if this app is launched on F-droid also. People like me who don't use play store will be really happy.

**Comment by avipars** [comment]
It is on IzzySoft, another repository that can be added to the f-droid app

https://apt.izzysoft.de/fdroid/index/apk/com.michaldrabik.showly_oss

**Reply by itsluminous** [reply]
but the source repo for that one is by [1RandomDev](https://github.com/1RandomDev) and his/her repo is a fork of this repo, so no guarantee that it will have no modifications.
It would be great if @michaldrabik could publish this package directly in fdroid or izzyondroid.

**Reply by avipars** [reply]
You are correct, I was looking at the wrong repo... if @michaldrabik  gives his okay, we can submit it to f-droid or izzyonandroid ... might be an issue in the current state as it seems to rely on firebase and google play services.
